assets/agenda/components/AgendaListItemIcons.jsx

```diff
@@ -87,6 +103,19 @@ class AgendaListItemIcons extends React.Component {
                     </div>
                 )}
 
+                {subject.length !== 0 && (
+                    <div>
+                        {subject.map((item, index) => (
+                            <span
+                                key={index}
```

Comment:
Using the array `index` as the `key` is not advisable as it's not a direct unique key to the item itself, it is better to use `item.qcode` in this case.